### PR TITLE
Fix to enable default dashboards

### PR DESCRIPTION
**Pull request by markoan**
Default daemon scripts won't load default dashboard because Cube needs to be run from installed directory.

### CODE REVIEW

File: templates/cube-collector [hunk 1]
```diff
@@ -17,6 +17,7 @@ NODE_PREFIX="%NODE_PREFIX%"
 PIDFILE="/var/run/cube/collector.pid"
 APP_DIR="/usr/bin"
 APP_BIN="env"
+NODE_DIR="$NODE_PREFIX/lib/node_modules/cube"
 NODE_SCRIPT="$NODE_PREFIX/lib/node_modules/cube/bin/collector.js"
```
Comment (lmammino) [inline templates/cube-collector]: Can you change the `NODE_SCRIPT` to use the `NODE_DIR` variable?

File: templates/cube-evaluator [hunk 2]
```diff
@@ -17,6 +17,7 @@ NODE_PREFIX="%NODE_PREFIX%"
 PIDFILE="/var/run/cube/evaluator.pid"
 APP_DIR="/usr/bin"
 APP_BIN="env"
+NODE_DIR="$NODE_PREFIX/lib/node_modules/cube"
 NODE_SCRIPT="$NODE_PREFIX/lib/node_modules/cube/bin/evaluator.js"
```
Comment (lmammino) [inline templates/cube-evaluator]: Same as above

**Comment by lmammino** [comment]
:+1:
Seem pretty good to me. 

If you can fix the commented lines I will test it and merge tomorrow (or at least on saturday).

Thanks a lot @markoan I really appreciate contribution on my work!

**Comment by markoan** [comment]
All done!
:+1:

**Comment by lmammino** [comment]
Did it! Thanks again @markoan :wink:

**Pull request merged**